pkg/skr/azureredisinstance/createKcpRedisInstance.go

```diff
@@ -0,0 +1,74 @@
+package azureredisinstance
+
+import (
+	"context"
+
+	cloudcontrolv1beta1 "github.com/kyma-project/cloud-manager/api/cloud-control/v1beta1"
+	cloudresourcesv1beta1 "github.com/kyma-project/cloud-manager/api/cloud-resources/v1beta1"
+	"github.com/kyma-project/cloud-manager/pkg/composed"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func createKcpRedisInstance(ctx context.Context, st composed.State) (error, context.Context) {
+	state := st.(*State)
+	logger := composed.LoggerFromCtx(ctx)
+
+	if state.KcpRedisInstance != nil {
+		return nil, nil
+	}
+
+	azureRedisInstance := state.ObjAsAzureRedisInstance()
+
+	state.KcpRedisInstance = &cloudcontrolv1beta1.RedisInstance{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      azureRedisInstance.Status.Id,
+			Namespace: state.KymaRef.Namespace,
+			Labels: map[string]string{
+				cloudcontrolv1beta1.LabelKymaName:        state.KymaRef.Name,
+				cloudcontrolv1beta1.LabelRemoteName:      azureRedisInstance.Name,
+				cloudcontrolv1beta1.LabelRemoteNamespace: azureRedisInstance.Namespace,
```

Comment:
Set these fields in annotations to avoid potential >63char names error (label can store max 63 characters)

Reply:
Those are moved to annotations.